Lib/test/test_peg_generator/test_c_parser.py

```diff
@@ -8,6 +9,10 @@
 from test.support import os_helper
 from test.support.script_helper import assert_python_ok
 
+_pyflags_nodist = sysconfig.get_config_var('PY_CFLAGS_NODIST') or ""
+if sysconfig.get_config_var('PGO_PROF_USE_FLAG') in _pyflags_nodist:
```

Comment:
```suggestion
if sysconfig.get_config_var('PGO_PROF_USE_FLAG') is not None and sysconfig.get_config_var('PGO_PROF_USE_FLAG') in _pyflags_nodist:
```

Reply:
`sysconfig.get_config_var('PGO_PROF_USE_FLAG')` can be None (chec the windows output).

Reply:
I made a small variation on that idea.